sdk/communication/azure-communication-email/setup.py

```diff
@@ -47,7 +47,7 @@
     url='https://github.com/Azure/azure-sdk-for-python',
     keywords="azure, azure sdk",  # update with search keywords relevant to the azure service / product
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```

Comment:
Just to confirm, do you support 'Programming Language :: Python :: 3.11',?

And do you reply on 

'msrest>=0.7.1',
'azure-common~=1.1',
'azure-mgmt-core>=1.3.2,<2.0.0',

It looks to me these are mgmt plane dependencies.

Just want to confirm.

Reply:
If you don't have any platform specific code (it seems to me this is your case), you support 3.11.